src/getStyles.js

```diff
@@ -1,4 +1,4 @@
-// @ts-check
+// @ts-nocheck
```

Comment:
@qwerty541, did you temporarily disable ts here because there are too many errors to fix?

Reply:
There is no errors inside this file. And i am sure that i did not changed this thing manually. Idk what happened, i revert this change.

Reply:
Nice catch :+1: